### ISSUE TITLE
FE search and display polygon for specific region

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#195]
## User story
1. As a User
2. I want to search for specific regions
3. So that I get to display data for those regions

## Acceptance criteria
* Search for regions (e.g.: Bavaria) for now only state (Bundesland) and county (Landkreis) wide
* Display polygon of regions

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
